docs/coding/financial-accounting.md

```diff
@@ -9,11 +9,11 @@ about your accounts as part of a type system.
 
 ## Building Intuition with Two Simple Examples
 
-If you have an outstanding loan and owe a bank $100, is your balance $100 or -$100? Conversely, if
-you have $200 in your bank account, is the balance $200 or -$200?
+If you have an outstanding loan and owe a bank `100`, is your balance `100` or `-100`? Conversely, if
+you have `200` in your bank account, is the balance `200` or `-200`?
```

Comment:
I removed the currencies from this documentation as it takes away from the core concept being conveyed.